README.md

```diff
@@ -309,6 +309,37 @@ new OrFilter(
 );
 ```
 
+#### Filtering by native date fields
+
+The Storyblok API allows filtering by publish or updated dates.
+Those fields could not be filtered by `gt_date` or `lt_date` operations and have dedicated parameters in the request.
+
+Supported date parameters are:
+* `published_at_gt`
+* `published_at_lt`
+* `first_published_at_gt`
+* `first_published_at_lt`
+* `updated_at_gt`
+* `updated_at_lt`
+
+```php
+use Storyblok\Api\StoriesApi;
+use Storyblok\Api\StoryblokClient;
+use Storyblok\Api\Domain\Value\QueryParameter\DateQueryParameter;
+use Storyblok\Api\Domain\Value\QueryParameter\QueryParameterCollection;
+use Storyblok\Api\Request\StoriesRequest;
+
+$client = new StoryblokClient(/* ... */);
+
+$storiesApi = new StoriesApi($client);
+$response = $storiesApi->all(new StoriesRequest(
+    language: 'de',
+    queryParameterCollection: new QueryParameterCollection([
+        new DateQueryParameter(DateQueryParameter::PUBLISHED_AT_GT, '<YYYY-MM-DD H:i>')
+    ])
```

Comment:
I think this is too generic and it s hard to understand what query collection is meant to be. 

I would rather update the Request objects to follow the DDD pattern. 

Maybe you could introduce ValueObjects for PublishedAt etc... with a DateTimeImmutable and an enum as second parameter with lt and gt as cases. 

This should increase DX 

And i am aware that not all query params are supported yet. But thats a great opportunity to evolve this package.

Reply:
removed collection